integration-test/pom.xml

```diff
@@ -244,7 +244,88 @@
         </executions>
       </plugin>
     </plugins>
-
   </build>
-
+  <profiles>
+    <profile>
+      <id>v2</id>
```

Comment:
Can you explain why we need this `v2` profile with duplicate plugin config?

Reply:
Cloud environments will break from the minikube flow here and not need the pre-integration-test phase to run at all. It's not strictly needed, and I'm happy to change it if we have a way to invoke the integration-test step without running the `pre-integration-test` step.

cc @echarles